backend/src/Conduit/Infrastructure/Slug.cs

```diff
@@ -5,7 +5,7 @@ namespace Conduit.Infrastructure
     // https://stackoverflow.com/questions/2920744/url-slugify-algorithm-in-c
     public static class Slug
     {
-        public static string? GenerateSlug(this string? phrase)
+        public static string? GenerateSlug(this string? phrase, int articleId)
```

Comment:
Why wouldn't we use `GenerateSlug` as an extension method for `Article` instead of `string?`, i.e. of its title. In this case you'd get `string slug = article.GenerateSlug()`

Reply:
Done.